src/components/autocomplete/js/highlightController.js

```diff
@@ -0,0 +1,25 @@
+(function () {
+  'use strict';
+  angular
+      .module('material.components.autocomplete')
+      .controller('MdHighlightCtrl', MdHighlightCtrl);
+
+  function MdHighlightCtrl ($scope, $element, $interpolate) {
+    var term = $element.attr('md-highlight-text'),
+        text = $interpolate($element.text())($scope.$parent);
```

Comment:
Since this does not create a new scope, why use `$scope.$parent` ? Wouldn't it reference the wrong scope (taking into account that the element will be transcluded) ?